landkreise/helper.py

```diff
@@ -32,6 +40,47 @@ def extract_status_date_directregex(text, regexmatch, input_date_format, match,
 def clear_text_of_ambigous_chars(text):
     return text.replace("\xa0", " ").replace("\r","\n")
 
+def get_status(text,occurrence=0):
```

Comment:
I think my changes are similar but affect different methods... I'll take time tomorrow to merge my code with yours. for today I'm done with coding.